api/src/main/java/com/eleks/academy/pharmagator/controllers/advices/PharmacyControllerAdvice.java

```diff
@@ -0,0 +1,85 @@
+package com.eleks.academy.pharmagator.controllers.advices;
+
+import com.eleks.academy.pharmagator.exceptions.ObjectNotFoundException;
+import com.eleks.academy.pharmagator.exceptions.UniqueConstraintViolation;
+import com.eleks.academy.pharmagator.validation.ValidationErrorResponse;
+import com.eleks.academy.pharmagator.validation.Violation;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+@ControllerAdvice
+public class PharmacyControllerAdvice {
+
+    @ExceptionHandler(value = {ObjectNotFoundException.class})
+    protected ResponseEntity<Map<String, Object>> handle(ObjectNotFoundException e) {
+
+        Map<String, Object> responseBody = new HashMap<>();
+
+        responseBody.put("errorMessage", e.getMessage());
+
+        responseBody.put("status", "Not found");
+
+        responseBody.put("statusCode", 404);
+
+        return ResponseEntity.badRequest().body(responseBody);
```

Comment:
Please use notFound() instead of badRequest()

Reply:
Fixed